reframe/frontend/statistics.py

```diff
@@ -264,7 +280,14 @@ def print_failure_report(self, printer, rerun_info=True):
                              f" -p {r['environment']} --system "
                              f"{r['system']} -r'")
 
-            printer.info(f"  * Reason: {r['fail_reason']}")
+            mssg = r['fail_reason']
```

Comment:
```suggestion
            msg = r['fail_reason']
```

Reply:
Also I would rather check against the `r['fail_info']['exc_type']` or the `exc_value`, which is more precise.